src/routes/expocamera.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { Button, Text, StyleSheet, Image, View, TouchableOpacity } from 'react-native';
+import { Camera, Permissions } from 'expo';
+
+import colors from '../assets/colors.js'
+import metrics from '../themes/metrics.js'
+
+import CameraButtonSVG from '../components/CameraButtonSvg';
+import LargeText from '../components/LargeText';
+import Navbar from '../components/Navbar'
+
+class CameraExample extends React.Component {
+  state = {
+    hasCameraPermission: null,
+    type: Camera.Constants.Type.back,
+    image: null
+  };
+
+  async componentWillMount() {
+    const { status } = await Permissions.askAsync(Permissions.CAMERA);
+    this.setState({ hasCameraPermission: status === 'granted' });
+  }
+
+  snap = async () => {
+    if (this.camera) {
+      this.camera.takePictureAsync()
+        .then(photo => this.setState({ image: photo }))
+    }
+  }
+
+  retakePicture = () => {
+    this.setState({image:null})
+  }
+
+  render() {
+    const { hasCameraPermission } = this.state;
+    const { goBack } = this.props.navigation;
+
+    const cameraView = (
+      <View style={{
+        flex: 1,
+        justifyContent: 'space-between',
+      }}>
+        <Camera
+          ref={ref => { this.camera = ref; }}
+          style={{ flex: 1 }} type={this.state.type}>
+          <View
+            style={{
+              position: 'absolute', 
+              flexDirection: 'row',
+              backgroundColor: 'black',
+              height: 90,
+              width: '100%',
+              justifyContent: 'flex-start',
+              bottom: 0,
+            }}>
+            <TouchableOpacity
+            style={{
+              width: '33%',
+              alignSelf: 'flex-end',
+              justifyContent: 'center',
+              alignItems: 'center',
+              height: 90,
+            }}
+            onPress={() => goBack()}>
+            <LargeText
+              style={{ alignSelf: 'center', textAlign: 'center', fontWeight: 'bold', color: 'white', margin: 0 }} text="Cancel"/>
+            </TouchableOpacity>
+              <TouchableOpacity
+              style={{
+                width: '33%',
+                alignSelf: 'flex-end',
+                justifyContent: 'center',
+                alignItems: 'center',
+                height: 90,
```

Comment:
I'd like to see these styles in a stylesheet. It's hard to read and refactor if it's inline.